CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - added a cheat to increase the game speed (#135)
 - added a matrix stack overflow error check and message if GetRoomBounds runs infinitely (#506)
 - added ability to turn off trex collision (#437)
+- added the ability to make new Pierre replace an old active (runaway) Pierre
```

Comment:
Also can you update this entry to include the issue number when that's opened? Also this line can probably be added to the README too (without the issue number, check out the other entries in it)

Reply:
Changelog and readme updated